ksql-serde/src/main/java/io/confluent/ksql/serde/util/SerdeProcessingLogMessageFactory.java

```diff
@@ -30,17 +31,30 @@ private SerdeProcessingLogMessageFactory() {
   public static Supplier<SchemaAndValue> deserializationErrorMsg(
       final Throwable exception,
       final Optional<byte[]> record) {
+    return deserializationErrorMsg(
+        exception,
+        record,
+        ProcessingLogConfig.getInstance()
```

Comment:
This type is only called from the deserialisers - so we don't need a static `getInstance` for this. The deserializer can grab the processing config out of the `configure` method, right?